frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx

```diff
@@ -152,6 +166,7 @@ const BundleDetail = ({
         revalidateOnMount: true,
         refreshInterval: refreshInterval,
         onSuccess: (response) => {
+            setFetchingMetadata(false);
```

Comment:
@epicfaace it's here:)

Reply:
Wait, why do we call setFetchingMetadata after call fetcherContents? Shouldn't we call setFetchingMetadata only after calling fetcherMetadata?

Reply:
Yeah just cleaned up everything